### DISCUSSION TITLE
vailant ecoTec Plus

### DISCUSSION THREAD

**Discussion by dirk99** [john30/ebusd#597] [General]
Hallo,
is it possible to query the Ebus which queries are possible? I would like to be able to control my heating with night setback as well as read out the values energy consumption. I have only the heating and no external controller. I get some values via BAI00, but the ebusd also finds a device EMM00. I have already linked all csv files to 15emm.csv, but I don't get any values but error messages for all values.
with ebusctl info I get the following messages:

address 00: master #1
address 03: master #11
address 08: slave #11, scanned "MF=Vaillant;ID=BAI00;SW=0704;HW=7603", loaded "vaillant/bai.308523.inc", "vaillant/08.bai.csv"
address 10: master #2
address 15: slave #2, scanned "MF=Vaillant;ID=EMM00;SW=0104;HW=8503", loaded "vaillant/15.emm.csv
address 31: master #8, ebusd
address 36: slave #8, ebusd

**Comment by john30** [comment]
`ebusctl find -a`
